frontend/src/pages/Admin/Components/Broadcast/ManageBroadcasts/Card/Card.jsx

```diff
@@ -84,13 +83,6 @@ export function Card(props) {
     "December",
   ];
 
-  const sanitizedContent = DOMPurify.sanitize(props.project.content);
-
-  const truncatedContent =
```

Comment:
This was good no? A separate method for truncation? Can you just modify the logic to use 250 characters here itself?